.eslintrc.js

```diff
@@ -42,43 +42,80 @@ module.exports = {
     },
   ],
   rules: {
-    // Possible Errors
-    // http://eslint.org/docs/rules/#possible-errors
+    // ESLint built-in rules
+    // http://eslint.org/docs/rules
+    'accessor-pairs': 'error',
+    'array-callback-return': 'error',
+    'arrow-parens': ['error', 'always'],
+    'arrow-spacing': ['error', { before: true, after: true }],
+    'block-spacing': 'error',
+    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    'comma-dangle': ['error', 'only-multiline'],
+    'comma-spacing': 'error',
+    'comma-style': 'error',
+    'computed-property-spacing': 'error',
+    'constructor-super': 'error',
+    'dot-location': ['error', 'property'],
+    'dot-notation': 'error',
+    'eol-last': 'error',
+    eqeqeq: ['error', 'smart'],
```

Comment:
Is there a reason some of these are `'` quoted and others not? If not, can we pick one style and be consistent?

Reply:
I definitely prefer it be consistent and intend to introduce a lint rule requiring that (if someone doesn't beat me to it). Was going to wait for this to land first to keep the pull requests minimal and focused. By waiting for this to land first, we avoid a merge conflict.

Reply:
I'm guessing they're quoted as needed for object properties.

Reply:
Yeah, I don't like that and would prefer that they all be quoted for consistency if one needs to be quoted. But that's just a preference. My plan is to try to figure out what the most common practice is in the code base, and then enforce that. If it's what's going on here, then so be it.